libusb/os/sunos_usb.c

```diff
@@ -0,0 +1,1259 @@
+/*
+ *
+ * Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hi,
     Corporate attorney agreed to change the copyright text to "Copyright (c) 2016 Oracle and/or its affiliates."

Thanks,
Lei Chen

On Mon 2016-Mar-28 17:36 PM Ludovic Rousseau wrote:
>
> In libusb/os/sunos_usb.c <https://github.com/libusb/libusb/pull/177#discussion_r57557821>:
>
> > @@ -0,0 +1,1259 @@
> > +/*
> > + *
> > + * Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
>
> Please do *not* use "All rights reserved."
>
> I would prefer something like (as suggested by the LGPL license itself):
>
> |Copyright (C) 2016 leilchen (use your real name and email)
> |
>
> "You should also get your employer (if you work as a programmer) or your school, if any, to sign a "copyright disclaimer" for the library, if necessary."
> See https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html#SEC4
>
> Thanks
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/libusb/libusb/pull/177/files/85ec215f03229fc1e571d3a802e49ff0f40eef06#r57557821>
>